### DISCUSSION TITLE
Fermentrack Temperature Incorrect

### DISCUSSION THREAD

**Discussion by StodgyWaif** [mp-se/gravitymon#44] [Q&A] [answered]
Hello.
I just installed GravityMon and it seems to be working well but I noticed the temperature recorded in Fermentrack is 30 degrees F or more higher than what the GravityMon web server page shows. Is this a push configuration error?
Thanks for any advice or information.

EDIT:
Okay I just figured out that GravityMon is detecting the temperature as 68F but it's sending it to Fermentrack as 68C.
Any idea how I can correct this?

**Comment by mp-se** [comment]
I will need to check this further. Dont know if the issue is in this software or fermentrack.

**Comment by mp-se** [comment, marked as answer]
Looks like I had a spelling error in the format defintion. The temp format should be "temp_units" and not "temp-units"... If you change the format to this it should work. Since fermentrack could not find the format type it defaults to celsius

{"name" : "${mdns}", "ID": "${id}", "token" : "${token}", "interval": ${sleep-interval}, "temperature": ${temp}, "temp_units": "${temp-unit}", "gravity": ${gravity}, "angle": ${angle}, "battery": ${battery}, "rssi": ${rssi}, "corr-gravity": ${corr-gravity}, "gravity-unit": "${gravity-unit}", "run-time": ${run-time} }
